<ui version="4.0" >
 <class>FindDlg</class>
 <widget class="QWidget" name="FindDlg" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>197</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Find</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>5</number>
   </property>
   <property name="spacing" >
    <number>5</number>
   </property>
   <item row="7" column="2" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1" >
    <widget class="QCheckBox" name="replaceChk" >
     <property name="text" >
      <string>Replace</string>
     </property>
     <property name="checked" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="2" >
    <widget class="QCheckBox" name="matchCaseChk" >
     <property name="text" >
      <string>Match case</string>
     </property>
    </widget>
   </item>
   <item rowspan="5" row="1" column="3" >
    <layout class="QVBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="findBtn" >
       <property name="text" >
        <string>Find</string>
       </property>
       <property name="default" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelBtn" >
       <property name="text" >
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="1" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Find</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2" >
    <widget class="QCheckBox" name="backwardChk" >
     <property name="text" >
      <string>Backward</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2" >
    <widget class="QCheckBox" name="regexpChk" >
     <property name="text" >
      <string>Regular expression</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2" >
    <widget class="QComboBox" name="findCmb" >
     <property name="editable" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="2" >
    <widget class="QComboBox" name="replaceCmb" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="editable" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>findCmb</tabstop>
  <tabstop>replaceChk</tabstop>
  <tabstop>replaceCmb</tabstop>
  <tabstop>matchCaseChk</tabstop>
  <tabstop>backwardChk</tabstop>
  <tabstop>regexpChk</tabstop>
  <tabstop>findBtn</tabstop>
  <tabstop>cancelBtn</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>replaceChk</sender>
   <signal>toggled(bool)</signal>
   <receiver>replaceCmb</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>50</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel" >
     <x>176</x>
     <y>58</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
